lib/primitive/test/spec/Cardano/Wallet/Primitive/CollateralSpec.hs

```diff
@@ -669,7 +669,7 @@ instance Arbitrary TxOut where
 
 spec :: Spec
 spec = do
-    parallel $ do
+    do
```

Comment:
Mb one `do` suffice? :)

Reply:
parallel do

Reply:
> Mb one `do` suffice? :)

Haha, nice find. 😆

Reply:
Fixed in 39206885d288e321a118622e95d9c7fe29262d4e.